frontend/src/utils/helpers/getLayoutBoxes/getLayoutBoxes.ts

```diff
@@ -26,9 +26,6 @@ const getLayoutBoxes = ({
   if (!wrapRef.current) {
     return {};
   }
-  // For multiple pinned participants with no screenshare we make all large tiles landscape
-  const shouldMakeLargeTilesLandscape =
-    layoutProps.pinnedSubscriberCount > 1 && !layoutProps.sessionHasScreenshare;
```

Comment:
Consider keeping this but changing it to:
`const shouldMakeLargeTilesLandscape = !layoutProps.sessionHasScreenshare;`